packages/local_auth/local_auth/CHANGELOG.md

```diff
@@ -1,7 +1,40 @@
-## NEXT
+## 2.0.0
 
+* Migrates plugin to federated architecture.
 * Adds OS version support information to README.
-* Switches over to default method implementation in new platform interface.
+* BREAKING CHANGE: Deprecated method `authenticateWithBiometrics` has been removed
+  Use `authenticate` instead.
+* BREAKING CHANGE: Parameters of `authenticate` have been changed.
+  
+  Example:
+```dart  
+// Old way of calling `authenticate`.
+Future<bool> authenticate(
+  localizedReason: 'localized reason',
+  useErrorDialogs: true,
+  stickyAuth: false,
+  androidAuthStrings: const AndroidAuthMessages(),
+  iOSAuthStrings: const IOSAuthMessages(),
+  sensitiveTransaction: true,
+  biometricOnly: false,
+);
+// New way of calling `authenticate`.
+Future<bool> authenticate(
+  localizedReason: 'localized reason',
+  authMessages: const <AuthMessages>[
+    IOSAuthMessages(),
+    AndroidAuthMessages()
+  ],
+  options: const AuthenticationOptions(
+    useErrorDialogs: true,
+    stickyAuth: false,
+    sensitiveTransaction: true,
+    biometricOnly: false,
+  ),
+);
+```
+
```

Comment:
We should add another breaking change highlighting that `strong` and `weak` have been added to the possible `getAvailableBiometrics` results, and that applications checking specific types should be updated to handle those as well.